packages/contracts/contracts/common/Common.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+/// @title Common
+/// @notice Contract that contains common things for all the contracts
+contract Common {
```

Comment:
not sure if it makes much of a difference with bytecode size but usually for shared errors or events one would use an interface rather than a contract